tests/utilities_t/test_asyncbulkcall.py

```diff
@@ -61,6 +61,22 @@ def test_delay_tiny(self):
         result = run_async(self.execute, execution, count=2048)
         self.assertEqual(result, [(i, i) for i in range(2048)])
 
+    def test_restart(self):
+        """Test that calls work after pausing"""
+        run_async(self.check_restart)
+
+    async def check_restart(self):
+        bunch_size = 4
+        # use large delay to only trigger on size
+        execution = AsyncBulkCall(CallCounter(), size=bunch_size // 2, delay=256)
+        for repeat in range(4):
```

Comment:
I think it would be nice to write `range(bunch_size)` instead of `range(4)`

Reply:
Thanks for pointing that out! Those are actually two unrelated `4`s, but I can totally see how that is confusing. I've changed it to another number to avoid the association.